core/src/main/scala/com/raphtory/internals/components/cluster/OrchestratorComponent.scala

```diff
@@ -44,6 +44,28 @@ abstract class OrchestratorComponent(conf: Config) extends Component[ClusterMana
       }
     }
 
+  protected def establishPartition(
+      component: String,
+      graphID: String,
+      clientID: String,
+      IDManager: IDManager,
+      func: (String, String, Config, IDManager) => Unit
```

Comment:
Does `func` need to be abstract. Not sure, but I don't see any scenario where we might want to call something different from `deployPartitionService`